webthing/server.py

```diff
@@ -341,6 +342,51 @@ def check_origin(self, origin):
         """Allow connections from all origins."""
         return True
 
+    def update(self):
```

Comment:
It doesn't look like `update()` is used anywhere.

Reply:
It can indeed be removed. I originally started using one update method, but then realized a Thing has separate notifications for an Action, Event and Property.